Automattic/wp-calypso
# Reader iFrames: Allows resizing iframes from WP sites.

#### Changes proposed in this Pull Request
~~I have allowed `allow-scripts` in the sandboxed iFrame so that it has the chance to send a message with the contents height. I am pretty sure that this is a security risk, I have created this PR just to kick off discussions.~~ discussed here https://github.com/Automattic/wp-calypso/pull/54410#discussion_r667830934

What has changed:
- If there are embeds from WordPress sites they will get resized to proper height
- Also, I they will get 100% of width
- and the blockquotes that used to hold the Title / link are removed since the Embed Title is now clickable.

If we eventually don't want to add `allow-scripts` in the sandbox, we have no way (?) of finding / applying the correct height. In that case, we can resort to having fixed height iFrames and some CSS which would apply a fading overlay on the bottom of the iFrames, we would lose though the bottom part of the embed
![](https://cln.sh/bQTSAU+)

#### Testing instructions



On a site you follow:
- Copy - paste `https://azcir.org/news/2021/06/14/504-plans-favor-wealthy-white-students-disabilities/` in your editor
- Save the post
- Visit the Calypso Reader and open this post
- Note that the iFrame gets resized to fit all the content
- Resizing the browser should resize the iFrames

Before | After
-------|------
<img width="751" alt="SS 2021-07-12 at 17 44 53" src="https://user-images.githubusercontent.com/12430020/125307629-e5eb2a00-e338-11eb-912e-670fad3da45a.png">|<img width="764" alt="SS 2021-07-12 at 17 45 54" src="https://user-images.githubusercontent.com/12430020/125307787-074c1600-e339-11eb-8a22-bfed1c8942af.png">




Fixes #45105
